uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/JolokiaJMXNavigatorTest.java

```diff
@@ -0,0 +1,138 @@
+
+	@RequirementRestriction
+	public static RequirementMatcher getRequirementMatcher() {
+		return new CamelExampleRequirementMatcher("camel-example-spring-boot", "2.20.1");
+	}
+	
+	/**
+	 * Prepares test environment
+	 */
+	@Before
+	public void defaultSetup() {
+		log.info("Deleting Error Log.");
+		LogView errorLog = new LogView();
+		errorLog.open();
+		errorLog.deleteLog();
+	}
+
+	/**
+	 * <p>
+	 * Test tries to create new <i>Jolokia connection</i> in JMX Navigator and access route nodes.
+	 * </p>
+	 * <ul>
+	 * <li>Create a new connection in JMX Navigator</li>
+	 * <li>New connection... --> <i>Jolokia Connection</i></li>
+	 * <li>Provide the name you want</li>
+	 * <li>Use Jolokia URL from Console View (by default <i>http://127.0.0.1:8778/jolokia</i>)</li>
+	 * <li>Select <i>Do NOT verify SSL Certificates</i></li>
+	 * <li>Click Finish</li>
+	 * <li>Expand User-Defined Connections --> double-click on the connection to activate it</li>
+	 * <li>Try to work with it (accessing nodes, perform available operations, ...)</li>
+	 * </ul>
+	 * 
+	 * @throws Exception
+	 */
+	@Test
+	public void testJolokiaAcccessingCamelRoute() throws Exception {
+
+		JolokiaConfiguration conf = camelExample.getConfiguration().getJolokiaConfiguration();
+
+		log.info("Try to open/activate 'JMX Navigator' view");
+		FuseJMXNavigator jmxView = new FuseJMXNavigator();
+		jmxView.open();
+		jmxView.activate();
+		jmxView.clickNewConnection();
+
+		log.info("Try to create a new jolokia connection");
+		JMXNewConnectionWizard connectionWizard = new JMXNewConnectionWizard();
+		assertTrue("Wizard 'Create JMX connection' was not opened properly", connectionWizard.isOpen());
+		connectionWizard.selectConnection(ConnectionType.JOLOKIA);
+		connectionWizard.next();
+		connectionWizard.setTextConnectionName(conf.getName());
+		connectionWizard.setTextJolokiaURL(conf.getUrl());	
+		if(!conf.isVerifySSL())
+			connectionWizard.toggleDoNOTVerifySSLCertificatesCHB(true);
+		connectionWizard.finish();
+
+		String connectionName = conf.getName();
+		log.info("Try to connect to the new jolokia connection");
+		jmxView.connectTo("User-Defined Connections", connectionName);
+		new WaitUntil(new JMXConnectionIsAvailable("User-Defined Connections", connectionName + "[Connected]"));
+
+		log.info("Try to access route components via new Jolokia connection");
+		assertJMXNode("User-Defined Connections", connectionName, "MBeans", "java.util.logging", "Logging", "LoggerNames");
+		jmxView.collapseAll();
+		assertJMXNode("User-Defined Connections", connectionName, "Camel", "SampleCamel", "Endpoints", "stream", "out");
+		jmxView.collapseAll();
+		assertJMXNode("User-Defined Connections", connectionName, "Camel", "SampleCamel", "Routes", "route1",
+						"timer:hello?period={{timer.period}}", "Transform transform1", "stream:out");
+		jmxView.collapseAll();
+		assertTrue("There are some errors in Error Log", LogGrapper.getPluginErrors("fuse").size() == 0);
```

Comment:
a LogChecker has been created, not sure it is merged yet. If not merged yet, that's fine to have it like that for now. Otherwise, please use it.